libvalkeycluster/vkutil.h

```diff
@@ -30,26 +30,26 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef __HIUTIL_H_
-#define __HIUTIL_H_
+#ifndef __VKUTIL_H_
```

Comment:
Let's move the files to ../src/ in the same PR? Renaming the file multiple times makes it even harder to track the git history.

Reply:
Moved renamed files to src/
Makefiles and so needs to be merged with hiredis/libvalkey..so Ill wait with them so sync with @michael-grunder